custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -219,35 +219,35 @@
             "visibilityCondition": "model.forecasting_style == 'customize_algorithms'"
         },
         {
-            "name": "naive_model_activated",
-            "label": "- Baseline",
-            "description": "Based on the trivial.identity/trivial.mean/seasonal_naive GluonTS models",
+            "name": "trivial_identity_model_activated",
+            "label": "- Trivial Identity",
+            "description": "Based on the trivial.identity GluonTS models",
             "type": "BOOLEAN",
             "defaultValue": true,
             "mandatory": true,
             "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
         },
         {
-            "name": "naive_model_method",
-            "label": "  ↳ method",
-            "type": "SELECT",
+            "name": "trivial_identity_model_kwargs",
+            "type": "MAP",
+            "label": "  ↳ Parameters",
+            "description": "",
+            "visibilityCondition": "model.forecasting_style == 'customize_algorithms' && model.trivial_identity_model_activated",
+            "mandatory": false,
+            "defaultValue": {"num_samples": 100}
```

Comment:
"num_samples – Number of samples to include in the forecasts. Not that the samples produced by this predictor will all be identical." ([gluon doc](https://ts.gluon.ai/api/gluonts/gluonts.model.trivial.identity.html))
num_samples is useless for trivial.identity, we should set by default for the user and remove the "↳ Parameters" param for trivial.identity